i18n/i18n.jsx

```diff
@@ -161,8 +161,9 @@ export function getLanguages() {
     if (!config.AvailableLocales) {
         return getAllLanguages();
     }
-
-    return config.AvailableLocales.split(',').filter((l) => languages[l]).map((l) => languages[l]);
+    const availables = {};
+    config.AvailableLocales.split(',').filter((l) => languages[l]).map((l) => Object.assign(availables, {[l]: languages[l]}));
```

Comment:
I think this can be achieved with, without needed to loop twice throw the array

```suggestion
    return config.AvailableLocales.split(',').reduce((result, l) => { if(languages[l]) { result[l] = languages[l]} return result; }, {});
```

Reply:
Thanks @enahum! Your suggestion is smart 👍 
I applied your suggestion with modifications for style error.